packaging/tests/tas-restapi/src/test/java/org/alfresco/rest/categories/DeleteCategoriesTests.java

```diff
@@ -0,0 +1,102 @@
+package org.alfresco.rest.categories;
+
+import org.alfresco.rest.RestTest;
+import org.alfresco.rest.model.RestCategoryModel;
+import org.alfresco.utility.data.RandomData;
+import org.alfresco.utility.model.FolderModel;
+import org.alfresco.utility.model.SiteModel;
+import org.alfresco.utility.model.TestGroup;
+import org.alfresco.utility.model.UserModel;
+import org.testng.annotations.BeforeClass;
+import org.testng.annotations.Test;
+
+import static org.alfresco.utility.report.log.Step.STEP;
+import static org.springframework.http.HttpStatus.BAD_REQUEST;
+import static org.springframework.http.HttpStatus.CREATED;
+import static org.springframework.http.HttpStatus.FORBIDDEN;
+import static org.springframework.http.HttpStatus.NOT_FOUND;
+import static org.springframework.http.HttpStatus.NO_CONTENT;
+
+public class DeleteCategoriesTests extends RestTest {
+
+    private UserModel user;
+
+
+    @BeforeClass(alwaysRun = true)
+    public void dataPreparation() throws Exception
+    {
+        STEP("Create a user");
+        user = dataUser.createRandomTestUser();
+    }
+
+    /**
+     * Check we can delete a category.
+     */
+    @Test(groups = {TestGroup.REST_API})
+    public void testDeleteCategory()
+    {
+        RestCategoryModel aCategory = createCategory();
+        restClient.authenticateUser(dataUser.getAdminUser()).withCoreAPI().usingCategory(aCategory).deleteCategory();
+        restClient.assertStatusCodeIs(NO_CONTENT);
```

Comment:
It might be worth also asserting that the category has gone (via a GET request).